### PR TITLE
Add GCS input to Akamai

### LINKED ISSUE #4291: [GCS Input] Add GCS to Cloudflare and Akamai
With the introduction of a new Google Cloud Storage input, users can now easily ingest events via GCS. The primary use case is 
to ingest logs from custom sources via GCS, however the following products natively support transmission of events to GCS. Our GCS input should be added to the relevant integration on our side accordingly. 

- [Akamai](https://techdocs.akamai.com/datastream2/docs/stream-google-cloud)
- [Cloudflare Log Push ](https://developers.cloudflare.com/logs/get-started/enable-destinations/google-cloud-storage/)

### PR DESCRIPTION
**Pull request by ShourieG**
## Type of change
- Enhancement

## What does this PR do?

Adds support for GCS input to the Akamai package.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues

- Closes #4291 

## Screenshots
Screenshot 2022-11-30 at 2 28 58 PM

### CODE REVIEW

File: packages/akamai/data_stream/siem/agent/stream/gcs.yml.hbs [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{{#if collect_gcs_logs}}
```
Comment (efd6) [inline packages/akamai/data_stream/siem/agent/stream/gcs.yml.hbs:1]: Does this need an `{{/if}}`?

File: packages/akamai/data_stream/siem/agent/stream/gcs.yml.hbs [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+{{#if bucket_timeout}}
+bucket_timeout: {{bucket_timeout}}
+{{/if}}
+{{#if buckets}}
+buckets:
+{{buckets}}
+{{/if}}
+{{#if tags}}
+tags:
+{{#each tags as |tag i|}}
```
Comment (efd6) [inline packages/akamai/data_stream/siem/agent/stream/gcs.yml.hbs:30]:
```suggestion
{{#each tags as |tag|}}
```

Comment (ShourieG) [reply packages/akamai/data_stream/siem/agent/stream/gcs.yml.hbs:30]: I'm not clear on why **i** is removed here . Do they have the same meaning ?

File: packages/akamai/data_stream/siem/agent/stream/gcs.yml.hbs (hunk 1 above)
Comment (ShourieG) [reply packages/akamai/data_stream/siem/agent/stream/gcs.yml.hbs:1]: yes, I think there should be an option for a toggle

File: packages/akamai/data_stream/siem/agent/stream/gcs.yml.hbs (hunk 2 above)
Comment (ShourieG) [reply packages/akamai/data_stream/siem/agent/stream/gcs.yml.hbs]: I have updated it, but can you explain the reason behind removing it .

Comment (efd6) [reply packages/akamai/data_stream/siem/agent/stream/gcs.yml.hbs]: The `i` is not needed. It is the index into the array and is never referenced. ref: https://handlebarsjs.com/guide/builtin-helpers.html#each and a playground snippet: https://handlebarsjs.com/playground.html#format=1&currentExample=%7B%22template%22%3A%22No%20second%20label%3A%5Cn%7B%7B%23each%20people%20as%20%7Cp%7C%7D%7D%5Cn%20%20%20%7B%7Bp%7D%7D%5Cn%7B%7B%2Feach%7D%7D%5Cn%5CnWith%20second%20label%3A%5Cn%7B%7B%23each%20people%20as%20%7Cp%20i%7C%7D%7D%5Cn%20%20%20%7B%7Bi%7D%7D.%20%7B%7Bp%7D%7D%5Cn%7B%7B%2Feach%7D%7D%5Cn%22%2C%22partials%22%3A%5B%5D%2C%22input%22%3A%22%7B%5Cn%20%20people%3A%20%5B%5Cn%20%20%20%20%5C%22Yehuda%20Katz%5C%22%2C%5Cn%20%20%20%20%5C%22Alan%20Johnson%5C%22%2C%5Cn%20%20%20%20%5C%22Charles%20Jolley%5C%22%2C%5Cn%20%20%5D%2C%5Cn%7D%5Cn%22%2C%22output%22%3A%22No%20second%20label%3A%5Cn%20%20%20Yehuda%20Katz%5Cn%20%20%20Alan%20Johnson%5Cn%20%20%20Charles%20Jolley%5Cn%5CnWith%20second%20label%3A%5Cn%20%20%200.%20Yehuda%20Katz%5Cn%20%20%201.%20Alan%20Johnson%5Cn%20%20%202.%20Charles%20Jolley%5Cn%22%2C%22preparationScript%22%3A%22%2F%2F%20Handlebars.registerHelper('loud'%2C%20function(string)%20%7B%5Cn%2F%2F%20%20%20%20return%20string.toUpperCase()%5Cn%2F%2F%20%7D)%3B%5Cn%22%2C%22handlebarsVersion%22%3A%224.7.7%22%7D

File: packages/akamai/data_stream/siem/agent/stream/gcs.yml.hbs (hunk 1 above)
Comment (efd6) [reply packages/akamai/data_stream/siem/agent/stream/gcs.yml.hbs:1]: So can the if be closed?

Comment (ShourieG) [reply packages/akamai/data_stream/siem/agent/stream/gcs.yml.hbs]: I have removed the if, the toggle switch here did not make sense.

**Review by efd6 — APPROVED** [review]

**Pull request merged**